### ISSUE TITLE
Force-open list doesn't work (reported from a user)

### ISSUE THREAD

**Issue by piroor** [clear-code/ieview-we#1]
> The addon work in manually mode (click on open the page in IE) but not with the list of webstites which i want they open with IE.

log:

```
          IndexedDB UnknownErr: ActorsParent.cpp:598  (source inconnue)
UnknownError  IndexedDB.jsm:259:18
[object DOMError]  ExtensionParent.jsm:1353
[object DOMError] ExtensionParent.jsm:1322
Avertissement : « nsIOService::NewChannel() » est obsolète, veuillez utiliser « nsIOService::NewChannel2() »  bootstrap.js:51:16
db is undefined ExtensionParent.jsm:1319
ieview-we: Sending:  Object { command: "read-mcd-configs" }  common.js:14:40
db is undefined  ExtensionParent.jsm:1319
ieview-we: Configs[storage] load: responded from others undefined  common.js:14:40
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist.  ExtensionCommon.jsm:306
IndexedDB UnknownErr: ActorsParent.cpp:598  (source inconnue)
UnknownError  IndexedDBHelper.jsm:71:12
ieview-we: Failed to read MCD configs:  Error: An unexpected error occurred  common.js:14:40
ieview-we: force list:  Array [ "https://www.dealabs.com/" ]  common.js:14:40
IndexedDB UnknownErr: ActorsParent.cpp:598  (source inconnue)
UnknownError  aboutHome.js:131:20
[Exception... "Component returned failure code: 0x80004005 (NS_ERROR_FAILURE) [nsIURI.host]"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "JS frame :: resource://gre/modules/WebRequest.jsm :: canModify :: line 842"  data: no] (source inconnue)
L’évènement « key » n’est pas disponible pour certaines dispositions de clavier : key="r" modifiers="accel,alt" id="key_toggleReaderMode"  browser.xul
L’évènement « key » n’est pas disponible pour certaines dispositions de clavier : key="i" modifiers="accel,alt,shift" id="key_browserToolbox"  browser.xul
ieview-we: onBeforeRequest Object { requestId: "1", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: 1, type: "main_frame", timeStamp: 1505296230736, frameId: 0, parentFrameId: -1 }  common.js:14:40
ieview-we: onBeforeRequest Object { requestId: "2", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: -1, type: "main_frame", timeStamp: 1505296231332, frameId: 0, parentFrameId: -1 }  common.js:14:40
ieview-we: Configs[storage] onMessage: Configs:load Object { type: "Configs:load" } Object { id: "ieview-we@clear-code.com", frameId: 55, url: "moz-extension://9ec32192-749e-4179-…", extensionId: "ieview-we@clear-code.com", contextId: "14-0", tab: Object }  common.js:14:40
ieview-we: Configs[storage] load  common.js:14:40
ieview-we: Configs[storage]  => already loaded  common.js:14:40
ieview-we: Configs[storage] load: responded from others Object { values: Object, lockedKeys: Object }  common.js:14:40
TelemetryStopwatch: key "TELEMETRY_PENDING_LOAD_MS" was already initialized  TelemetryStopwatch.jsm:343
TelemetryStopwatch: requesting elapsed time for nonexisting stopwatch. Histogram: "TELEMETRY_PENDING_LOAD_MS", key: "null" TelemetryStopwatch.jsm:363
ieview-we: onBeforeRequest Object { requestId: "3", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: -1, type: "main_frame", timeStamp: 1505296284737, frameId: 0, parentFrameId: -1 }  common.js:14:40
ieview-we: onBeforeRequest Object { requestId: "4", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: 4, type: "main_frame", timeStamp: 1505296289351, frameId: 0, parentFrameId: -1 }  common.js:14:40
ieview-we: Configs[storage] set: debug = false  common.js:14:40
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. ExtensionCommon.jsm:306
ieview-we: Configs[storage] broadcast updated config: debug = true (locked: false)  common.js:14:40
ieview-we: Configs[storage] onMessage: Configs:updated Object { type: "Configs:updated", key: "debug", value: true, locked: false } Object { id: "ieview-we@clear-code.com", frameId: 55, url: "moz-extension://9ec32192-749e-4179-…", extensionId: "ieview-we@clear-code.com", contextId: "14-0", tab: Object }  common.js:14:40
ieview-we: Configs[storage] successfully saved Object { debug: true }  common.js:14:40
Unchecked lastError value: Error: Could not establish connection. Receiving end does not exist. ExtensionCommon.jsm:306
IndexedDB UnknownErr: ActorsParent.cpp:598  (source inconnue)
UnknownError  indexed-db.js:58:9
ieview-we: onBeforeRequest Object { requestId: "6", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: 4, type: "main_frame", timeStamp: 1505296328777, frameId: 0, parentFrameId: -1 }  common.js:14:40
ieview-we: onBeforeRequest Object { requestId: "7", url: "https://www.dealabs.com/", originUrl: undefined, documentUrl: undefined, method: "GET", tabId: -1, type: "main_frame", timeStamp: 1505296338257, frameId: 0, parentFrameId: -1 }
```

**Comment by piroor** [comment]
Additional info:

> In background.js line 34, it's not  open(aDetails.url); but launch(aDetails.url); i think. Can you confirm?

**Comment by WoPlin** [comment]
I have the same issue, the listed sites are not working with Firefox 55.0.3 / FirefoxESR 52.3.0

**Comment by piroor** [comment]
Fixed by 64ffc3d1d1ba618894c3d6bed9e706d6e39d544b
